pyqmc/optimize_excited_states.py

```diff
@@ -463,7 +471,7 @@ def find_move_from_line(
     # print("offdiagonal_energy", energy)
     # print("norm",np.einsum('ijj->i', (overlap-1)**2 ))
     cost = (
-        np.sum(energy.diagonal(axis1=1, axis2=2), axis=1)
+        np.sum(np.multiply(energy_weights, energy.diagonal(axis1=1, axis2=2)), axis=1)
```

Comment:
Shouldn't you use energy_weights*energy.diagonal(axis1=1, axis2=2) instead of `multiply`?